### PR TITLE
Remove vm mutex 

### PR DESCRIPTION
**Pull request by akokoshn**

### CODE REVIEW

File: _studio/mfx_lib/shared/include/mfx_frames.h [hunk 1]
```diff
@@ -77,31 +77,28 @@
     class clExtTasks1
     {
     public:
-        clExtTasks1()
+        clExtTasks1():
+            m_mGuard()
```
Comment (dvrogozh) [inline _studio/mfx_lib/shared/include/mfx_frames.h]: emmm. why? this is useless and should be done automatically by compiler.

File: _studio/mfx_lib/shared/include/mfx_frames.h [hunk 2]
```diff
@@ -167,32 +160,29 @@
     class clExtTasks2
     {
     public:
-        clExtTasks2()
+        clExtTasks2():
+            m_mGuard()
```
Comment (dvrogozh) [inline _studio/mfx_lib/shared/include/mfx_frames.h]: same

File: _studio/mfx_lib/shared/include/mfx_frames.h (hunk 1 above)
Comment (mgonchar) [reply _studio/mfx_lib/shared/include/mfx_frames.h]: agree

**Comment by mgonchar** [comment]
@dvrogozh could you look again?

**Pull request merged**